hack/update-codegen-crds.sh

```diff
@@ -17,24 +17,27 @@
 set -o errexit
 set -o nounset
 set -o pipefail
-# set -o xtrace
+set -o xtrace
 
-if [[ -z "${MAKELEVEL:-}" ]]; then
-    echo 'You must invoke this script via make'
+if [[ -z "${CONTROLLER_GEN:-}" ]]; then
+    echo "You must either set CONTROLLER_GEN to the path to controller-gen or invoke via make"
     exit 1
 fi
 
+REPO_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)
+
 # Update generated CRD YAML
-cd pkg/apis
-../../${CONTROLLER_GEN} \
-    crd \
-    rbac:roleName=manager-role \
-    webhook \
-    paths="./..." \
-    output:crd:artifacts:config=../../config/crds
-cd -
-
-for CRD in ./config/crds/*.yaml; do
+(
+    cd "${REPO_ROOT}/pkg/apis"
+    "../../${CONTROLLER_GEN}" \
+        crd \
+        rbac:roleName=manager-role \
+        webhook \
+        paths="./..." \
+        output:crd:artifacts:config="${REPO_ROOT}"config/crds
+)
```

Comment:
Nothing later depends on what is the current working directory at this point, if I follow correctly; that makes the subshell unnecessary.

Reply:
removed